docs/src/docs/pykitops/mlflow.md

```diff
@@ -0,0 +1,49 @@
+---
+description: Use KitOps PyKitOps Python library to automate ModelKit creation with MLFlow.
+---
+# MLFlow and KitOps ModelKits
+
+Many KitOps users automate the creation of secure and tamper-proof [ModelKits](../modelkit/intro.md) for each experiment run in MLFlow. This gives the team and organization a library of models that they can reproduce on any infrastructure.
+
+## Install PyKitOps Python Library
+
+PyKitOps is a community provided Python library for simplifying the creation and management of ModelKits from code. First install PyKitOps into the python interpreter used for by MLFlow for experiment runs:
+
+```sh
+pip install kitops
+```
+
+## Downloading the Run Artifacts
+
+After the experiment run is complete, download the artifacts to a local directory - this is typically after the current `with mlflow.start_run() as cur_run` execution is complete.
+
+```py
+artifact_location = mlflow.artifacts.download_artifacts(cur_run.info.run_id)
+```
+
+This will return a local directory of the artifacts (referred to as `artifact_location` below).
+
+## Packaging the Artifacts as a ModelKit
+
+The `ModelKitManager` is then used to pack and upload the artifacts as a ModelKit.
+
+In the following code:
+`artifact_location` = the directory where the experiment run artifacts were saved
+`name` = the name of the ModelKit as it will be displayed in the [Kitfile](../kitfile/kf-overview.md)
```

Comment:
@javisperez can I still link things like this with relative paths?

Reply:
this works good on my machine! but yeah, relatives are now confusing :(